helm/aphw-ddi-api/templates/_container.yaml

```diff
@@ -1,6 +1,6 @@
 {{- define "aphw-ddi-api.container" -}}
-livenessProbe: {{ include "ffc-helm-library.http-get-probe" (list . .Values.livenessProbe) | nindent 4}}
-readinessProbe: {{ include "ffc-helm-library.http-get-probe" (list . .Values.readinessProbe) | nindent 4}}
+livenessProbe: {{ include "ffc-helm-library.http-get-probe" (list . .Values.livenessProbe) | nindent 4 }}
```

Comment:
I thought these were changed in a previous PR (as well as the extra env var in docker-compise-api-tests/yaml? Have we not picked up the latest or did a revert branch lose the changes?